packages/react-components/react-aria/stories/src/useTypingAnnounce/useTypingAnnounceDescription.md

```diff
@@ -0,0 +1,7 @@
+`useTypingAnnounce` takes a reference to an element (which should be an `input`, `textarea`, or `contenteditable`) and returns a `typingAnnounce` function with the same paramters as the `announce` function returned by `useAnnounce`.
```

Comment:
>which should be an `input`, `textarea`, or `contenteditable`

Do you think it's worth enforcing this with a type?

Reply:
Hm, I think I lean towards not. There might be some other use case for announcing while a user is typing (e.g. Dropdown, where you type to jump to an item, but it isn't actually editable). That's kind of a stretch, but I don't really see a benefit in making it a code restriction.